modules/ob-configurable-fields-in-build.adoc

```diff
@@ -0,0 +1,50 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-openshift-builds.adoc
+
+:_content-type: REFERENCE
+[id="ob-configurable-fields-in-build_{context}"]
+= Configurable fields in build
+
+You can use the following fields in your `Build` custom resource (CR):
+
+.Fields in the `Build` CR
+[options="header"]
+|===
+
+| Field | Presence | Description
+
+| `apiVersion` | Mandatory | Specifies the API version of the resource, for example, `shipwright.io/v1alpha1`.
+
+| `kind` | Mandatory | Specifies the type of the resource, for example, `Build`.
+
+| `metadata` | Mandatory | Denotes the metadata that identifies the custom resource definition instance, for example, the name of the `Build` resource.
+
+| `spec.source` | Mandatory | Denotes the location of the source code, for example, a Git repository or source bundle image.
+
+| `spec.strategy` | Mandatory | Denotes the name and type of the build strategy.
```

Comment:
Unsure, but is this "build" referencing to the strategy for the `Build` resource?

Reply:
yes